tests/jerry/es2015/class-inheritance-bounds.js

```diff
@@ -0,0 +1,28 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var g = Array.bind (0, 1, 2, 3)
```

Comment:
nitpicking: why is this test file named "bounds"?

Reply:
The reason is that implicit class constructors are really similar to bound functions. But let's rename it only to `-bound.js`, hence it only includes one test case.